### PR TITLE
fix(VToolbar): don't allow toolbar to grow in flex column

### PR DESCRIPTION
**Pull request by mactanxin**
## Description

Fix VToolbar height with prop `fullscreen`

## Motivation and Context

It doesn't show height properly

## How Has This Been Tested?

visually and yarn test

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
<template>
  <v-app>
    <v-container>
      <v-btn @click="dialog = true">show dialog</v-btn>
      <v-dialog
        
        v-model="dialog"
        @click:outside="cancel"
        @keydown.esc="cancel"
      >
        <v-card width="500">
          <v-toolbar
            dark
            color="primary"
          >
            <v-btn
              icon
              dark
              @click="cancel"
            >
              <v-icon>mdi-close</v-icon>
            </v-btn>
            <v-toolbar-title>Settings</v-toolbar-title>
            <v-spacer />
            <v-toolbar-items>
              <v-btn
                dark
                text
                @click="cancel"
              >
                Save
              </v-btn>
            </v-toolbar-items>
          </v-toolbar>
          <v-card-title v-if="customTitle" class="mt-2">
            <v-icon left color="primary"> mdi-delete-outline</v-icon>
            title
          </v-card-title>
          <p class="d-flex justify-start mb-0 ml-7 text--secondary pr-6">
            Delete something...
          </p >
          <v-card-actions class="pb-4 mt-3 mr-3">
            <div class="flex-grow-1" />
            <v-btn
              variant="outlined"
              class="mr-4"
              @click="cancel"
            >
              Cancel
            </v-btn>
            <v-btn
              variant="outlined"
              color="primary"
              @click="confirm"
            >
              Confirm
            </v-btn>
          </v-card-actions>
        </v-card>
      </v-dialog>
    </v-container>
  </v-app>
</template>

<script setup>
  import { computed, ref } from 'vue'
  const dialog = ref(false)
  const customTitle = ref('Delete Dialog')
  const cancel = () => {
    dialog.value = false;
  }
</script>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by johnleider** [comment]
It's not immediately obvious to me what this is fixing. What are the steps to reproduce this issue on current next?

**Comment by KaelWD** [comment]
Probably #8606

**Comment by mactanxin** [comment]
@johnleider @KaelWD yes, actually shown in [Current V3 Card Fullscreen demo](https://next.vuetifyjs.com/en/components/dialogs/)

File: packages/vuetify/src/components/VToolbar/_variables.scss [hunk 1]
```diff
@@ -18,7 +18,7 @@ $toolbar-content-padding-x: 16px !default;
 $toolbar-content-padding-y: 4px !default;
 $toolbar-elevation: 0 !default;
 $toolbar-flat-elevation: 0 !default;
-$toolbar-flex: 1 1 auto !default;
+$toolbar-flex: 0 1 auto !default;
```
Comment (KaelWD) [inline packages/vuetify/src/components/VToolbar/_variables.scss:21]:
```suggestion
$toolbar-flex: none !default;
```

**Comment by erthi16** [comment]
We could just add the v-toolbar at line 89 in the v-dialog--scrollable block of the file VDialog.sass like for v-card__title and v-card__action.

**Pull request merged**